CI/remoteLayerCaculation.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+cd docs/private_tangle
+
+ROOT_TREE_LOCAL=`cat data/layers/layer.0.csv`
```

Comment:
When was this file created?

Reply:
in the `setupTests` phase - the merkle tree is calculated;
https://travis-ci.org/iotaledger/compass/jobs/514107326#L2358